### PR TITLE
Issue #2899254: Integrate editorial search with Search API module (6.x)

### PR DESCRIPTION
**Pull request by mtodor**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -138,6 +142,8 @@
         "drupal/thunder_fia": "^1.0",
         "drupal/token": "^1.0",
         "drupal/update_helper": "^1.0",
+        "drupal/search_api": "1.15",
+        "drupal/views_bulk_operations": "^2.4",
```
Comment (chrfritsch) [inline composer.json]: views_bulk_operations 3 is already out and recommended

File: composer.json [hunk 2]
```diff
@@ -74,6 +74,9 @@
             },
             "drupal/redirect": {
                 "Update view to met the Drupal 8.8 views schema": "https://www.drupal.org/files/issues/2019-12-23/3089389-6.patch"
+            },
+            "drupal/search_api": {
+                "Reset-tracker drush command functionality is misleading": "https://www.drupal.org/files/issues/2020-03-12/3119335.patch"
```
Comment (chrfritsch) [inline composer.json]: Patch needs adjustment

**Review by chrfritsch — CHANGES_REQUESTED** [review]

**Comment by dbosen** [comment]
We should add search to media tab and media browser(?)

**Comment by dbosen** [comment]
We need better test coverage for this.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -268,7 +269,7 @@ jobs:
         run: test-drupal-project run_tests
 
   test-performance:
-    needs: [build, test-max]
```
Comment (dbosen) [inline .github/workflows/test.yml:271]: Lets not forget to put that back

File: composer.json [hunk 4]
```diff
@@ -102,6 +102,7 @@
         "drupal/entity_reference_revisions": "^1.3",
         "drupal/field_group": "3.1",
         "drupal/focal_point": "^1.3",
+        "drupal/facets": "1.x-dev",
```
Comment (dbosen) [inline composer.json:105]: -dev? Oh boy... I guess, we need to change this to @dev now, that we have stability alpha

**Review by dbosen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**